src/app/components/WorkDetail/DefinitionList.jsx

```diff
@@ -17,13 +18,31 @@ export const labels = {
 /**
  * Build a definition list of elements from a bibliographic record provided
  * by Elastisearch.
+ *
  * @param {object} props
  */
 export const DefinitionList = (props) => {
+  /**
+   * onClick handler for browse searches.
+   *
+   * @param {object} event
+   * @param {string} query
+   * @param {string} field
+   */
+  const newSearchRequest = (event, query, field) => {
+    event.preventDefault();
+
+    props.dispatch(searchPost(query, field))
```

Comment:
In testing this doesn't seem to make a difference, but we might want to wrap the `query` here in quotes to force the API/ElasticSearch to only return exact matches. Generally this is what the `AND`ed queries will do, but for a simple subject with some common terms it _could_ have unintended results.

Reply:
Not sure if we can add quotes around the query there without another parameter signaling that the query is a "browse" query coming from a link rather than an ad-hoc query typed in the search box or URL. But seems like a decision for another PR to handle.